snippets/base/admin/fields.py

```diff
@@ -20,9 +20,11 @@ def clean(self, value):
 class JEXLChoiceField(ChoiceField):
     def __init__(self, attr_name, *args, **kwargs):
         self.attr_name = attr_name
+        self.jexl = '{attr_name} == {value}'
+        self.jexl = kwargs.pop('jexl', self.jexl)
```

Comment:
Also, since the `to_jexl` method below requires `attr_name` and `value` as placeholders, should we validate those exist here if `kwargs.pop` is not empty?

Could be a bit overkill, but would make the code semi self-documenting.